inject/src/main/java/io/micronaut/context/DefaultBeanContext.java

```diff
@@ -2078,7 +2080,7 @@ protected void initializeContext(
         if (CollectionUtils.isNotEmpty(parallelBeans)) {
             processParallelBeans(parallelBeans);
         }
-        ForkJoinPool.commonPool().execute(() -> beanDefinitionsClasses.forEach(p -> p.isReferenceEnabled(this)));
+        ForkJoinPool.commonPool().execute(() -> beanDefinitionsClasses.forEach(p -> p.getReferenceIfEnabled(this)));
```

Comment:
What is this line trying to do originally? It looks a bit odd.

Reply:
I think it's kicking off async loading of all bean definitions that were not initialized by the previous code